client/app/views/contact.coffee

```diff
@@ -167,11 +168,17 @@ module.exports = class ContactView extends ViewCollection
 
             @model.setN @contactName.getStructuredName()
             @model.set note: @notesfield.val()
+            @model.set 'bday', @$('.bday-input').val()
+            @model.set 'org', @$('.org-input').val()
+            @model.set 'title', @$('.title-input').val()
+            @model.set 'department', @$('.department-input').val()
+            @model.set 'url', @$('.url-input').val()
```

Comment:
url is a complex field / datapoint, same than email & tel (one contact can have multiple URLs). Is there a particular client you wanted compatibility with to make this change ?